assets/src/Components/StarterSites/Filters.js

```diff
@@ -119,6 +120,13 @@ const Filters = ( {
 						onlyProSites={ onlyProBuilders }
 						count={ counted.builders }
 					/>
+					{ ! tiobDash.isValidLicense && editor !== 'elementor' && (
+					<Notification
+						data={tiobDash.upsellNotifications['upsell_1'] }
+						editor={editor}
+					/>
+					)
+					}
```

Comment:
Please enable eslint formatting on save on your end or run `eslint --fix` on this file 🦭